### PR TITLE
NIAD-74 Parallel pipeline

### PR DESCRIPTION
**Pull request by petervdm**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -4,243 +4,323 @@ pipeline {
...
+                                            label: 'Obtaining the dynamodb table name used for the MHS sync/async state',
+                                            returnStdout: true,
+                                            script: "terraform output mhs_sync_async_table_name"
+                                        ).trim()
+                                    }
+                                }
+                            }
+                        }
+
+                        stage('Deploy SCR') {
```
Comment (mattd-kainos) [inline Jenkinsfile]: Not needed anymore?

File: docker-compose.yml [hunk 2]
```diff
@@ -2,7 +2,10 @@ version: '3'
 
 services:
   inbound:
-    image: local/mhs-inbound:${INBOUND_BUILD_TAG}
+    image: local/mhs-inbound:${BUILD_TAG}
+    # build:
```
Comment (mattd-kainos) [inline docker-compose.yml:6]: Is there a ticket in the backlog for what is still left to do?

**Review by mattd-kainos — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (petervdm) [reply docker-compose.yml:6]: Yes - https://gpitbjss.atlassian.net/browse/NIAD-154

File: Jenkinsfile (hunk 1 above)
Comment (petervdm) [reply Jenkinsfile]: No, removed.

**Review by grzegorz-robak — APPROVED** [review]

**Review by mattd-kainos — APPROVED** [review]

**Pull request merged**